docs/topics/Gateway.md

```diff
@@ -450,7 +463,7 @@ Sent on connection to the websocket. Defines the heartbeat interval that the cli
 | Field              | Type             | Description                                                     |
 | ------------------ | ---------------- | --------------------------------------------------------------- |
 | heartbeat_interval | integer          | the interval (in milliseconds) the client should heartbeat with |
-| \_trace            | array of strings | used for debugging, array of servers connected to               |
+| \_trace            | unspecified      | used for debugging, this is internal data and may change format |
```

Comment:
What about READY and RESUMED?

Reply:
Actually, I think the best thing is that I'm just going to undocument it -- it's an internal field.

Reply:
Then I would recommend creating a new field to replace it like `_timings` which jake suggested. That way the libraries shouldn't break. Keep the `_traces` as is and remove it from the docs.